examples/scope.rb

```diff
@@ -21,17 +21,23 @@
 # Example: Create a Scope with a simple description
 oneview_scope 'Scope1' do
   client my_client
-  api_version 600
```

Comment:
Since I am not able to add the comment on line#18, I am adding here..
please update the api version at Line#18 to 1200

Reply:
done